### DISCUSSION TITLE
Agenda and Expectations (Plus Gratitude)

### DISCUSSION THREAD

**Discussion by dgerding** [couchbaselabs/couchbase-efcore-provider#30] [General]
Hi Jeffry and Brant,
Very excited about the possibility of a Couchbase Lite EF Core provider. Do you have a rough estimate of when you anticipate being at something like a first version? I'm trying like crazy to make the abrupt move from "Realm" (ouch) to Couchbase Lite and an EF Core implementation would be a real win for me and plenty of other .net devs.

Fwiw, I am hoping, like many, to take an offline first/local first design assumption and hope as you move ahead with your EF Core Provider implementation you'll keep in mind that many apps require local first and offline ("rare sync to capella") design assumptions. 

Any thoughts or estimates appreciated and thanks for the work on this welcome resource.
Dave G

**Comment by jeffrymorris** [comment]
@dgerding -

There is a [GA](https://www.nuget.org/packages/Couchbase.EntityFrameworkCore/1.0.0) up on NuGet. Check it out and please provide any feedback.

Thanks,
Jeff
